lib/book_parser.py

```diff
@@ -106,19 +104,26 @@ def _get_paragraph_data(self, codec):
 
         # find all paragraphs (sections enclosed in html p tags) and their starting offset
         for node in re.finditer(PARAGRAPH_PAT, book_html):
-            # get plain text from paragraph and locations of letters from beginning of file
-            results = [(word.group(0)[1:-1].decode(codec), word.start(0))
-                       for word in re.finditer(PLAIN_TEXT_PAT, node.group(0))]
             word_loc = {'words': '', 'locs': [], 'char_sizes': []}
```

Comment:
i realized that when you looked in the book, it was messing up the paragraphs when you look at the places where our special list of names/words was listed.. it would print the wrong sections.. everything that was changed in this file was meant to fix that.. i found that when we parse the gunslinger by stephen king, it would parse incorrectly so i originally was trying to fix that issue but couldn't figure out what was there.. see #91 .. while i was working on that, i found this issue so i fixed it